servicetalk-http-api/src/main/java/io/servicetalk/http/api/DefaultBlockingStreamingHttpRequest.java

```diff
@@ -154,6 +154,18 @@ public BlockingStreamingHttpRequest setQueryParameters(String key, String... val
         return this;
     }
 
+    @Nullable
+    @Override
+    public String fragment() {
```

Comment:
There is no need to add override for "getter" in all classes that implement `HttpRequestMetaData`. It's enough to have it only in:
1. AbstractDelegatingHttpRequest
2. DefaultHttpRequestMetaData